CONTRIBUTING.md

```diff
@@ -301,6 +301,17 @@ To manually create a backport pull request, follow [the standard pull request pr
 - When the backport pull request is merged and you have the necessary privileges you can also replace the label `9.needs: port to stable` with `8.has: port to stable` on the original pull request.
   This way maintainers can keep track of missing backports easier.
 
+## How to request review of pull requests
+[pr-review-request]: #how-to-request-review-of-pull-requests
+
+Due to the large number of pull requests that need to be handled, some of them may be overlooked by the reviewer, may not receive feedback after completing the requested changes, or may be ignored by committers and remain unmerged for a long time. You can remedy this situation but notifying reviewers and committers through the following channels:
+
+- [NixOS Discourse: PRs ready for review](https://discourse.nixos.org/t/prs-ready-for-review/3032) for pull requests not yet receiving any reviews, already approved or made requested changes.
```

Comment:
I know this is used by many people, but this thread for me is an indicator that something is very wrong with our tooling. Can't we fix that instead of sending even more people to a yet another, different medium?